public/video-ui/src/components/VideoUpload/YoutubeUpload.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Icon from '../Icon';
+import { ManagedForm } from '../ManagedForm';
+import VideoUtils from '../../util/video';
+
+export default class YoutubeUpload extends React.Component {
+  state = { file: null };
+
+  setFile = event => {
+    if (!this.props.video) {
+      return;
+    }
+
+    if (event.target.files.length === 0) {
+      this.setState({ file: null });
+    } else {
+      this.setState({ file: event.target.files[0] });
+    }
+  };
+
+  render() {
+    const { video, startUpload } = this.props;
+
+    const canUploadToYouTube = VideoUtils.canUploadToYouTube(video);
+
+    return (
+      <div className="video__detailbox video__detailbox__assets">
+        <div className="video__detailbox__header__container">
+          <header className="video__detailbox__header">
+            Upload to YouTube
+          </header>
+        </div>
+        <div className="form__group">
+          <ManagedForm
```

Comment:
Do we get anything useful from the managed form here when we are not using the managed field component?

Reply:
good point! removed